sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -1930,11 +1930,17 @@ object QueryCompilationErrors {
   }
 
   def descPartitionNotAllowedOnTempView(table: String): Throwable = {
-    new AnalysisException(s"DESC PARTITION is not allowed on a temporary view: $table")
+    new AnalysisException(
+      errorClass = "FORBIDDEN_OPERATION",
+      messageParameters =
+        Array(toSQLStmt("DESC PARTITION"), "the temporary view", toSQLValue(table)))
```

Comment:
Please, use `toSQLId` instead of `toSQLValue`, see https://github.com/apache/spark/pull/36210

Reply:
Done.